### PR TITLE
Translation table library v2 improvements

### PR DESCRIPTION
**Pull request by sandrine-bailleux-arm**

### CODE REVIEW

File: lib/xlat_tables_v2/xlat_tables_common.c [hunk 1]
```diff
@@ -1,123 +0,0 @@
-/*
```
Comment (danh-arm) [inline lib/xlat_tables_v2/xlat_tables_common.c]: Need to retain this file but leave it empty for compatbility

Comment (danh-arm) [inline lib/xlat_tables_v2/xlat_tables_common.c]: Sorry, I've changed my mind. I forgot this was in xlat_tables_v2, which we don't guarantee compatibility for yet.

**Comment by davidcunado-arm** [comment]
jenkins: test this please

**Comment by sandrine-bailleux-arm** [comment]
Hi @masahir0y,

As part of this pull request, I had to modify the UniPhier platform Makefile to make sure it still builds properly. Could you please have a look and see whether you approve the changes? The specific commit I'm talking about is [this one](https://github.com/ARM-software/arm-trusted-firmware/pull/1035/commits/a9ad848ccf084998a17bbb2870903b0a686de2ef).

Regards,
Sandrine

**Pull request merged**